### ISSUE TITLE
Provide more helpful messages when running "clean" tasks with doit

### ISSUE THREAD

**Issue by gdemonet** [scality/metalk8s#1969]
**Component**: build

**Why this is needed**:

- When some files in the temporary `_build` directory are not known by the buildchain (either because they were added manually, or because they are remnants of previous builds on other versions), `doit clean` only outputs a message saying `canot remove (it is not empty) '<directory>'`
- Messages from `doit clean <task>` are not easy to read for humans, showing the internal identifiers of tasks (e.g. `_deploy_salt_tree:salt/make_tree`) for which the `clean` operation is executed

**What should be done**:

- Find some way of changing the formatting for `clean` messages
- Improve error messages when finding non-managed files (e.g. list them for the user to know if they would like to `rm -rf _build` or not)

**Implementation proposal** (strongly recommended):

TBD, though [the docs](https://pydoit.org/cmd_other.html#clean) don't mention anything out-of-the-box to replace the default formatter

**Test plan**: Manual, check that non-managed files are well displayed in case of `doit clean`.

**Comment by slaperche-scality** [comment]
> TBD, though the docs don't mention anything out-of-the-box to replace the default formatter

Could be worth discussing the matter with upstream :slightly_smiling_face:

**Comment by gdemonet** [comment]
> > TBD, though the docs don't mention anything out-of-the-box to replace the default formatter
> 
> Could be worth discussing the matter with upstream

Indeed, we'll look at our options and open an issue upstream to discuss it with them
